### ISSUE TITLE
Unable to select a server with VoiceOver enabled

### ISSUE THREAD

**Issue by sevapopov2** [AdguardTeam/AdGuardVPNForMac#444]
### Please answer the following questions for yourself before submitting an issue.

- [x] I checked the [knowledge base](https://adguard-vpn.com/kb/) and found no answer
- [x] I checked to make sure that this issue has not already been filed

### AdGuard VPN version

2.7.0.860

### Environment

```markdown
- OS: Monterey 12.7.6.
- Other network-level software:
Not applicable.
```

### Which DNS server do you use?

Default

### Issue Details

Preconditions:
VoiceOver enabled.
Steps to reproduce:
1. Launch the AdGuard-VPN app.
2. Navigate to the server in the list with CTRL+Option+Left/right arrow keys and Activate the selected option with CTRL+Option+Space command.

### Expected Behavior

The connection to the server happens after activating the server in the list

### Actual Behavior

Nothing happens.

### Screenshots

Not applicable.

### Additional Information

The reason why elements don't work with VoiceOver as expected is because servers don't have the correct role.
Currently, text role is used.
Role button is required to work with VoiceOver as expected.

**Comment by IaroslavKhvorostenko** [comment]
Hello @sevapopov2! 
Try the latest version on nightly update at this link: https://agrd.io/mac_vpn_nightly

**Comment by sevapopov2** [comment]
Hello,
Thank you for your answer!

> Try the latest version on nightly update at this link: https://agrd.io/mac_vpn_nightly

Unfortunately, after instaling the provided version and launching it there are no any items defined by VoiceOver, the window is empty when navigating with commands.
I tried to restart app and Mac but it didn't work.

**Comment by IaroslavKhvorostenko** [comment]
Hello @sevapopov2! I have rechecked the information.

Support for CTRL+Option+Left/Right and CTRL+Option+Space command is currently unavailable, and this is the expected behavior at the moment. Support for these commands is planned for v.2.9.0.

Earlier I suggested you install the Nightly version. Updates are released more frequently in this version, so you can continue using it.

**Comment by sevapopov2** [comment]
> Earlier I suggested you install the Nightly version. Updates are released more frequently in this version, so you can continue using it.

Was the link you provided the link to nightly version?
If so, it didn't work and currently I cannot use any items in the app at all.
If I have to switch to nightly version manually, I cannot do that unfortunately since this process is not accessible with VoiceOver.
How can I switch to nightly version?
Thank you!

**Comment by IaroslavKhvorostenko** [comment]
@sevapopov2 
Link to install the Nightly version: https://agrd.io/mac_vpn_nightly
